src/js/lineChartTimeSeriesLine.js

```diff
@@ -0,0 +1,146 @@
+/*
+Copyright 2016 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    // Mixin grade for time-series line
+
+    fluid.defaults("floe.chartAuthoring.lineChart.timeSeries.line", {
+        model: {
+            dataSet: []
+        },
+        lineOptions: {
+            interpolation: "linear"
+        },
+        selectors: {
+            chartLine: ".floec-ca-lineChart-chartLine"
+        },
+        styles: {
+            chartLine: "floe-ca-lineChart-chartLine"
+        },
+        listeners: {
+            "onDraw.drawChartLine": {
+                func: "{that}.drawChartLine"
+            }
+        },
+        modelListeners: {
+            dataSet: [{
+                funcName: "{that}.events.onDraw.fire",
+                excludeSource: "init"
+            }]
+        },
+        invokers: {
+            drawChartLine: {
+                funcName: "floe.chartAuthoring.lineChart.timeSeries.line.drawChartLine",
+                args: ["{that}"]
+            }
+        }
+    });
+
+    floe.chartAuthoring.lineChart.timeSeries.line.drawChartLine = function (that) {
+        var dataSet = that.model.dataSet,
+            chartLineSelector = that.options.selectors.chartLine,
+            svg = that.svg;
+
+        // Bind data and keep a reference to the bound elements
+        that.chartLinePaths = svg.selectAll("path" + chartLineSelector)
+            .data(dataSet, function (d) {
+                return d.id;
+            });
+
+        floe.chartAuthoring.lineChart.timeSeries.line.updateChartLine(that);
```

Comment:
This should be moved over to our now-standard "namespaced, prioritised listeners" approach to allow this workflow to be customised